lemur/authorities/views.py

```diff
@@ -227,6 +228,11 @@ def post(self, data=None):
            :statuscode 403: unauthenticated
```

Comment:
nit, this can use update too, calling out 401 as unauthenticated and 403 as unauthorized/forbidden

Reply:
Thanks, good point. There are 53 other places in the code that define 403 as unauthenticated, which is never true as far as I am aware 😬 I won't touch those now, though.